### PR TITLE
ER-model

### PR DESCRIPTION
**Pull request by Johney9**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (stanislavgrujic) [inline .idea/vcs.xml]: This should go into the .gitignore

File: uml/er-model/ERModel.puml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+@startuml
```
Comment (stanislavgrujic) [inline uml/er-model/ERModel.puml]: I expected that you will create a full Java  Maven project which uses the `plantuml-maven-plugin` to generate the ER schema.
In that case, you would have hierarchy `src/main/resources/uml` to place these files.

**Comment by stanislavgrujic** [comment]
Could you please attach generated screenshot, it will be easier to do the review?

**Comment by Johney9** [comment]
Changed project to full Maven, using plantuml-maven-plugin to generate the files. Added .idea folder to .ignore. Also added the mysql model and dependencies to pom.xml

ermodel

**Comment by stanislavgrujic** [comment]
What does file `src/main/resources/sql/model/team_pomodoro_db_model.mwb` represent? If it is generated, it should not be commit in the source code control.

**Comment by Johney9** [comment]
It represents the mysql model from which the db schema is generated. Should I remove it?

**Comment by stanislavgrujic** [comment]
Which tool did you use to do it? Please remove the .bak, and this one depends on your answer. :)

**Comment by Johney9** [comment]
I used mysql workbench. Will remove .bak

**Comment by stanislavgrujic** [comment]
PNG files should not be commit and should not go into the `src` folder, but should go into `target` folder, with `.class` files

File: .gitignore [hunk 3]
```diff
@@ -20,3 +20,11 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+#IntelliJ IDE project folder
+src/test/
```
Comment (stanislavgrujic) [inline .gitignore]: Why would you ignore the test files?

File: .gitignore [hunk 4]
```diff
@@ -20,3 +20,11 @@
...
+/src/main/resources/sql/scripts/schema_create_script.sql
```
Comment (stanislavgrujic) [inline .gitignore]: Why is this file ignored, is it generated? If so, it should not go into `/src/main/resources` folder.

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                        <includes>
+                            <include>src/main/resources/uml/**/*.puml</include>
+                        </includes>
+                    </sourceFiles>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>net.sourceforge.plantuml</groupId>
+                        <artifactId>plantuml</artifactId>
+                        <version>8059</version>
```
Comment (stanislavgrujic) [inline pom.xml]: There is a dependency version mismatch between this one and the one used in the project dependencies definition. Please extract `plantuml.version` and use the same version in all places.

**Pull request merged**